arviz/plots/kdeplot.py

```diff
@@ -46,7 +46,7 @@ def plot_kde(values, values2=None, cumulative=False, rug=False, label=None, bw=4
         Keywords passed to the fill under the line (use fill_kwargs={'alpha': 0} to disable fill).
         Ignored for 2D KDE
     rug_kwargs : dict
-        Keywords passed to the rug plot. Ignored if rug=False or for 2D KDE
+        Keywords passed to the rug plot. Ignored if rug=False for 2D KDE
```

Comment:
the or is right in the sense that `rug_kwargs` will be ignored either if rug=False or if the plot is a 2D KDE

Reply:
My mistake, thank you for the review. Will make the change